tools/issue-labeler/src/IssueLabeler/IssueLabeler.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We should rename the folder, solution, and project files to "IssueLabelerService" or similar so that its clear.